src/components/Navigation/type.ts

```diff
@@ -0,0 +1,11 @@
+import * as H from 'history';
+
+export interface INavigationProps {
+  userHasAuthenticated: (isAuthenticated: boolean) => void;
+  isAuthenticated: boolean;
+  history: H.History;
+}
+
+export interface INavigationStates {
```

Comment:
Look like abandoned type?

Reply:
You're right! Currently, we don't use this Interface. I will fix it.